gobblin-temporal/src/main/java/org/apache/gobblin/temporal/workflows/metrics/TemporalEventTimer.java

```diff
@@ -104,7 +104,7 @@ public TemporalEventTimer create(String eventName) {
     public TemporalEventTimer createJobTimer() {
       TemporalEventTimer startTimer = create(TimingEvent.LauncherTimings.JOB_START);
       startTimer.stop(Instant.EPOCH); // Emit start job event containing a stub end time
-      return create(TimingEvent.LauncherTimings.JOB_COMPLETE, startTimer.startTime);
+      return create(TimingEvent.LauncherTimings.JOB_SUCCEEDED, startTimer.startTime);
```

Comment:
suggest a comment to convey that `SUCCEEDED` was chosen in favor of `COMPLETE`, to satisfy gaas' expectations